home.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CineOnline</title>
+    <link href="css/estilos.css" rel="stylesheet"/>
+</head>
+<body>
+
+<!--Cabecero de la página principal "HOME"-->
+<header>
+    <nav class="nav-fixed">
+        <img id="logo" src="imagenes/cine_adobespark (1).png" alt="Logo Cinemath"/>
+        <h3 id="id">CineOnline</h3>
+        <a href="home.html">Home</a>
+        <a href="movies.html">Películas</a>
+        <a href="home.html">Series</a>
+        <a href="home.html">Precios</a>
+        <a href="home.html">Contacto</a>
+        <a href="home.html" id="slg">Login</a>   
+    </nav>      
+</header>
+<!--Sección 1 de la página principal "HOME"-->
+    <section id="seccion1">
+        <h1 id="h1s1">Nosotros te ayudamos a encontrar las mejores películas, series y shows de televisión</h1>

```

Comment:
Mantener la consistencia en el lenguaje, si vamos a utilizar Español que todos los textos sean en Español.

```suggestion
        <h1 id="h1s1">Nosotros te ayudamos a encontrar las mejores películas, series y programas de televisión</h1>
```